### PR TITLE
Adds workshop recommendations to module 3

### PR DESCRIPTION
**Pull request by fcaroline2020**
### Common Content?

- [ ] Block/s

### Common Theme?

- [ ] Yes

Issue number: #issue-number

### Org Content?

Module | Sprint | Page Type | Block Type

## Checklist

- [ ] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [ ] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [ ] I have run my code to check it works
- [ ] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

File: org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md [hunk 1]
```diff
@@ -15,6 +15,18 @@ time=15
 name="Workshop"
 src="blocks/workshop"
 time="140"
+  [[blocks.nested.blocks]]
+    name="TBD [PD] (30 Mins)"
+    src="TBD"
+    time=0
+  [[blocks.nested.blocks]]
+    name="Debugging 3 [Tech] (60 Mins)"
```
Comment (illicitonion) [inline org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md:23]: Debugging React stuff seems quite ambitious? We probably want to remove this one?

**Review by illicitonion — COMMENTED** [review]
LGTM except for the TBDs breaking the build :) Thanks!

I filed https://github.com/CodeYourFuture/CYF-Workshops/pull/262 with a few minor copy edits to the workshops

File: org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md (hunk 1 above)
Comment (fcaroline2020) [reply org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md:23]: Where do you think this fits better in the curriculum or should we leave it out completely? 
Also is there a workshop off the top of your head that fits better here?

Comment (illicitonion) [reply org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md:23]: I think we should probably leave it out completely, it relies on material we don't teach at all... Maybe @SallyMcGrath has an idea of a good replacement :)

Comment (fcaroline2020) [reply org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md]: Removed and will pick up this topic over the weekend + open a new pull request

**Review by kfklein15 — APPROVED** [review]
looks good

File: org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md (hunk 1 above)
Comment (SallyMcGrath) [reply org-cyf-itp/content/data-groups/sprints/3/day-plan/index.md]: https://github.com/CodeYourFuture/CYF-Workshops/tree/main/components-2

https://github.com/CodeYourFuture/CYF-Workshops/tree/main/goblin-breakdown

**Pull request merged**